ui/app/src/components/ItemMenu/ItemMenu.tsx

```diff
@@ -240,15 +236,13 @@ export function ItemMenu(props: ItemMenuProps) {
             onCancel: closeConfirmDialog(),
             onOk: batchActions([
               closeConfirmDialog(),
-              showNewContentDialog({
-                open: true,
-                rootPath: path,
+              showChangeContentTypeDialog({
                 item,
-                type: 'change',
+                rootPath: getRootPath(item.path),
                 selectedContentType: item.contentTypeId,
                 onContentTypeSelected: batchActions([
-                  closeNewContentDialog(),
-                  changeContentType({ contentTypeId: item.contentTypeId, path })
+                  closeChangeContentTypeDialog(),
+                  changeContentType({ contentTypeId: item.contentTypeId, path: item.path })
```

Comment:
Can we have `newContentTypeId` and `prevContentTypeId`?

Reply:
yes